### DISCUSSION TITLE
connecting multiple nodes to a single node at once in GLSP

### DISCUSSION THREAD

**Discussion by arvind-bandi** [eclipse-glsp/glsp#1326] [Q&A]
Hi All,
I am working on GLSP diagram nodes. I have been trying to connect multiple nodes to a single  node.  I am selecting  all 3 nodes together and trying to connect them to the 4th one but I can only connect to one node(one-one). 
is there a way to connect multiple nodes at once ?
![Screenshot 2024-04-29 122012](https://github.com/eclipse-glsp/glsp/assets/168404075/c55f4ff9-1254-450e-8aab-37c0d7b2c175)

**Comment by planger** [comment]
With the default [edge creation tool](https://github.com/eclipse-glsp/glsp-client/blob/master/packages/client/src/features/tools/edge-creation/edge-creation-tool.ts), connecting multiple nodes at once is not possible unfortunately. This would have to be done with a customization of the editing tools that take the selection into account.
